apps/site/src/pages/account/api-keys-panel/api-keys-list.tsx

```diff
@@ -0,0 +1,201 @@
+
+    const { displayName, publicId } = data;
+    return {
+      apiKey: data,
+      matchingNewlyCreatedKey: newlyCreatedKeyIds.find((key) =>
+        key.includes(publicId),
+      ),
+      keyAction:
+        keyActionStatus?.publicId === publicId
+          ? keyActionStatus.action
+          : undefined,
+      renameApiKeyCard: (
+        <ApiKeyCard
+          onClose={dismissKeyAction}
+          defaultValue={displayName}
+          showDiscardButton
+          submitTitle="Rename key"
+          inputLabel="Rename your key"
+          onSubmit={async (newDisplayName) =>
+            renameApiKey(publicId, newDisplayName)
+          }
+        />
+      ),
+      revokeApiKeyCard: (
+        <RevokeApiKeyCard
+          onClose={dismissKeyAction}
+          displayName={displayName}
+          onRevoke={async () => revokeApiKey(publicId)}
+        />
+      ),
+    };
+  };
+
+  if (isMobile) {
+    return (
+      <Box>
+        {apiKeys.map((data, index) => (
+          <>
+            <MobileApiKeyItem
+              key={data.publicId}
+              {...generateApiKeyItemProps(data)}
+            />
+
+            <Box
+              key={`${data.publicId}-divider`}
+              sx={{
+                mt: 3,
+                mb: index < apiKeys.length - 1 ? 3 : 0,
+                borderTop: "1px solid",
+                borderColor: "gray.30",
+              }}
+            />
+          </>
+        ))}
+
+        {isCreatingNewKey && (
+          <Box mt={3}>
+            <Typography sx={{ mb: 2, fontWeight: 500, fontSize: 16 }}>
+              Create new key
+            </Typography>
+            {createKeyCard}
+          </Box>
+        )}
+      </Box>
+    );
+  }
+
+  return (
+    <Box>
+      <TableContainer>
+        <Table>
+          <TableHead
+            sx={{
+              th: {
+                pl: 0,
+                py: 1,
+                color: "gray.80",
+                fontSize: 14,
+              },
+            }}
+          >
+            <TableRow>
+              {apiKeys.length ? (
+                <>
+                  <TableCell sx={{ width: 130 }}>Name</TableCell>
+                  <TableCell>Token</TableCell>
+                  <TableCell>Last Used</TableCell>
+                  <TableCell>Created</TableCell>
+                  <TableCell />
+                </>
+              ) : (
+                <TableCell colSpan={5}>Choose a name</TableCell>
+              )}
+            </TableRow>
+          </TableHead>
+          <TableBody
+            sx={{
+              td: {
+                color: "gray.90",
+                pl: 0,
+                fontWeight: 500,
+              },
+            }}
+          >
+            {apiKeys.map((data) => (
+              <ApiKeyTableRow
+                key={data.publicId}
+                {...generateApiKeyItemProps(data)}
+              />
+            ))}
+            {isCreatingNewKey && (
```

Comment:
When this appears, it's causing other columns to resize. I was experiencing this with revoke/rename cards too but can't repro now.

Reply:
Wrapping the line after some width fixes this issue. Not sure about the underlying reason, but not sure if it's worth spending time figuring out as well. So this should fix this issue https://github.com/blockprotocol/blockprotocol/pull/1257/commits/31e29204a100a22303e53dd529c4fb9da33e66f2